invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/MobileCommunitiesItem.js

```diff
@@ -0,0 +1,65 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import _get from "lodash/get";
+import React from "react";
+import { Image } from "react-invenio-forms";
+import { Icon, Item, Label } from "semantic-ui-react";
+
+export const MobileCommunitiesItem = ({ result, index }) => {
+  const community_type = result.ui?.type?.title_l10n;
+  const visibility = result.access.visibility;
+  const isPublic = visibility === "public";
+  const visibilityColor = isPublic ? "green" : "red";
+  const visibilityText = isPublic
+    ? i18next.t("Public")
+    : i18next.t("Restricted");
+  const visibilityIcon = isPublic ? undefined : "ban";
+  return (
+    <Item key={index} className="mobile only">
```

Comment:
please set the community id as a key, instead of the array index (no need to pass it down through the component)